pyproject.toml

```diff
@@ -31,12 +31,8 @@ dependencies = [
   # (plugins.mkdocstrings.handlers.python.import)
   "frequenz-client-microgrid >= 0.4.0, < 0.5.0",
   "frequenz-channels >= 1.0.0-rc1, < 2.0.0",
-  "google-api-python-client >= 2.71, < 3",
-  "grpcio >= 1.60.1, < 2",
-  "grpcio-tools >= 1.60.1, < 2",
   "networkx >= 2.8, < 4",
   "numpy >= 1.26.4, < 2",
-  "protobuf >= 4.21.6, < 6",
   "pydantic >= 2.3, < 3",
   "timezonefinder >= 6.2.0, < 7",
   "tqdm >= 4.38.0, < 5",
```

Comment:
`tqdm` and `watchfiles` are unused dependencies as well, maybe they can go away too, along with these.

Reply:
Actually `timezonefinder` is not used anymore either (it was moved to the microgrid API). Added a new commit removing dependencies not used for a long time now.